1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,15 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let start = 0;
+  let toxicList = [];
+  while (toxicList.length < n) {
+    toxicList.push(start);
+    start += 2;
+  }
+  console.log(toxicList);
+
+  // TODO
```

Comment:
Perhaps need more testing / changing here to make it work

Reply:
thanks, updated.